contracts/Holdable.sol

```diff
@@ -0,0 +1,240 @@
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "./IHoldable.sol";
+
+contract Holdable is IHoldable, ERC20 {
+
+   using SafeMath for uint256;
+
+    struct Hold {
+        address issuer;
+        address origin;
+        address target;
+        address notary;
+        bool expires;
+        uint256 expiration;
+        uint256 amount;
+        HoldStatusCode status;
+    }
+
+    mapping(string => Hold) private holds;
+    mapping(address => uint256) private heldBalance;
+    mapping(address => address[]) private operatorsAuthorized;
```

Comment:
The operators could be saved in one variable with `mapping(address => mapping(address => bool)) private operators;`. Then you can simply check for `true` and `false` and don't need to work with an array